Fix grammar in the comments in TIA-MC1 and Instrebiteli drivers

sev- opened the pull request:
I went through the comments for TIA-MC1 and Istrebiteli drivers and fixed their grammar.

No functional changes were brought.

cuavas commented:
There are more spelling and grammar errors introduced in this PR, and changing C++ line comments to C style isn't helpful.  You've also changed the meaning of several comments in addition to changing grammar.

sev- commented:
Weird reaction. I am the original author of tiamc1 file, and yes, I wanted uniformity, as the latter additions introduced mix of the comment style.

Could you please clarify, where I changed meaning and where I introduced more spelling errors?

cuavas commented on src/mame/drivers/tiamc1.cpp:
```diff
@@ -486,35 +486,35 @@ ROM_START( gorodki )
 ROM_END
 
 /*
-  later (cost reduced ?) version of TIA-MC1 hardware, uses single PCB main board design,
```
"Later" is correct here

cuavas commented on src/mame/drivers/tiamc1.cpp:
```diff
@@ -331,7 +331,7 @@ static MACHINE_CONFIG_START( tiamc1, tiamc1_state )
 	/* video hardware */
 	MCFG_SCREEN_ADD("screen", RASTER)
-	MCFG_SCREEN_RAW_PARAMS(PIXEL_CLOCK, 336, 0, 256, 312, 0, 256)       // pixel clock and htotal comes from docs/schematics, the rest is guess (determined by undumped PROM)
+	MCFG_SCREEN_RAW_PARAMS(PIXEL_CLOCK, 336, 0, 256, 312, 0, 256)       /* pixel clock and htotal come from docs/schematics, the rest is a guesswork (originall specified by an undumped PROM) */
```
It should be "the rest is a guess" or "the rest is guesswork".  "the rest is a guesswork" is bad grammar.  Also, "originall" is a typo.

cuavas commented on src/mame/drivers/istrebiteli.cpp:
```diff
@@ -5,19 +5,19 @@
-      dumped PCB is early game version, have several bugs, possible test/prototype.
-      later version was seen in St.Petersburg arcade museum, CPU board have single 8Kx8 ROM.
+      dumped PCB is from an early game version which has several bugs, perhaps it is a test/prototype.
+      latter version was seen in the St.Petersburg arcade museum, CPU board has a single 8Kx8 ROM.
```
"later" is correct here

cuavas commented:
I've pointed out introduced errors with comments against the commit.

C++ line comments are better for notes at the end of lines of code because you can easily block-comment them out, and they're less visual clutter as they don't have an extra visual token at the end of the text.

My biggest gripe though was the introduction of grammatical and spelling errors by a PR that claimed to address the same.
